tests/src/Functional/InstallationTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace  Drupal\Tests\contenta_jsonapi\Functional;
+
+use Drupal\Component\Serialization\Json;
+use PHPUnit\Framework\TestCase;
+use GuzzleHttp\Client;
+/**
+ * Tests that installation finished correctly and known resources are available.
+ *
+ * @group ContentaInstaller
+ */
+class InstallationTest extends TestCase {
+
+    /**
+     * @var Client
+     */
+    private $httpClient;
+
+    /**
+     * @var string
+     */
+    private $baseUrl;
+
+    /**
+     * @var array
+     */
+    private $extras;
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+      $Url = getenv('WEB_HOST');
```

Comment:
Do you mind ensuring that the file follows the drupal coding style? aka. 2 spaces indentation etc. phpcbf should fix most of that for free for you.